### PR TITLE
Adding ruby

### PR DESCRIPTION
**Pull request by HenryLeongStat**
Professor, I add Ruby and following the context of other languages. Please take a look at it and feel free to make any change!

### CODE REVIEW

File: src/homepage/Running_SoS.ipynb [hunk 1]
```diff
@@ -561,6 +561,20 @@
...
+   "metadata": {},
+   "source": [
+    "### <a id=\"Ruby\"></a><img src=\"img/Ruby.png\" style=\"width:32pt;height:32pt;margin-right:15pt;\">Ruby"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use Ruby with SoS, you will need to install [iRuby](https://github.com/SciRuby/iruby) by following [iRuby Kernel homepage](https://github.com/SciRuby/iruby). After that, install [daru](https://github.com/SciRuby/daru) and [NMatrix](https://github.com/SciRuby/nmatrix)."
```
Comment (BoPeng) [inline src/homepage/Running_SoS.ipynb]: Install `daru` ... with command `gem install daru nmatrix`? This is standard command so I suppose we will not get into trouble for including them.

Comment (HenryLeongStat) [reply src/homepage/Running_SoS.ipynb]: Yup! Unlike kernels, including the instruction of installing library is OK...

File: src/documentation/Supported_Languages.ipynb [hunk 2]
```diff
@@ -2677,6 +2677,1147 @@
...
+    "  | Sequence (`list`, `tuple`, ...) |  |  `Array` |\n",
+    "  | `set` |  |  `Set` |\n",
+    "  | `dict` |  |  `Hash` |\n",
+    "  | `range` |  |  `Range` |\n",
+    "  | `numpy.ndarray` |  | `Array` |\n",
+    "  | `numpy.matrix` |  | `NMatrix` |\n",
+    "  | `pandas.Series` |  | `Hash` |\n",
+    "  | `pandas.DataFrame` |  | `Daru::DataFrame` |\n",
+    "\n",
+    "  Python objects in other datatypes are transferred as string `\"Unsupported datatype\"`."
```
Comment (BoPeng) [inline src/documentation/Supported_Languages.ipynb]: I would add, perhaps also in other places, something like:

as string "Unsupported datatype". Please [let us know](git hub issue tracker) if there is a natural corresponding data type in Ruby to convert this data type.

Comment (BoPeng) [reply src/documentation/Supported_Languages.ipynb]: There are too many such cases. Perhaps you can add a general comment before specific languages.

**Comment by BoPeng** [comment]
Merged. Thanks. https://vatlab.github.io/sos-docs/doc/documentation/Supported_Languages.html#Ruby-17

**Pull request merged**